feathr_project/feathr/_databricks_submission.py

```diff
@@ -85,31 +85,35 @@ def upload_or_get_cloud_path(self, local_path_or_http_path: str):
         elif src_parse_result.scheme.startswith('dbfs'):
             # passed a cloud path
             logger.debug(
```

Comment:
Should this be a warning? 

For error message, it's better to tell what is supported too.